lib/src/common/js_proxy.dart

```diff
@@ -145,11 +145,32 @@ dynamic dartValue(jsValue) {
       }
       _addDartInstance(jsValue, dartMap);
       return dartMap;
+    } else if (jsValue is JsFunction) {
+      return _dartType(jsValue);
```

Comment:
Yea I can move that up so it doesn't get caught in the check for JsObject as JsFunction descends from it.